<?xml version="1.0"?>
<launch>
    <env name="GSCAM_CONFIG" value="v4l2src device=/dev/video0 ! video/x-raw-yuv,framerate=30/1,width=1920,height=1080 ! ffmpegcolorspace " />
    <node pkg="gscam" type="gscam" name="mobius">
        <param name="camera_name" value="mobius" />
        <param name="camera_info_url" value="$(env HOME)/.ros/camera_info/head_camera.yaml" />
        <!-- <remap from="camera/image_raw" to="$(arg cam_name)/image_raw" /> -->
    </node>

    <node name="detector" pkg="detector" type="detector_node" output="screen">
        <rosparam file="$(find detector)/cfg/detector.yaml"/>
        <rosparam file="$(env HOME)/.ros/camera_info/head_camera.yaml"/>
        <remap from="image" to="/camera/image_raw"/>
        <remap from="bounding_boxes" to="bounding_boxes"/>
        <remap from="object_poses" to="object_poses"/>
        <remap from="undist_image" to="undist_image"/>
        <remap from="marked_image" to="marked_image"/>
    </node>

</launch>